missas/core/management/commands/postgres2sqlite.py

```
@@ -0,0 +1,221 @@
+    def order_tables_by_dependencies(self, tables, db_alias):
+        """
+        Order tables based on foreign key dependencies using topological sort.
+        Tables with no dependencies come first.
+        """
+        dependencies = {}
+        all_tables = set(tables)
+
+        for table in tables:
+            fk_relations = self.get_foreign_keys(table, db_alias)
+            deps = set()
+            for column_index, (ref_table, ref_column) in fk_relations.items():
```

Ruff (0.14.4)

Loop control variable `column_index` not used within loop body

Rename unused `column_index` to `_column_index`

(B007)